app/views/draws/show.html.erb

```diff
@@ -37,6 +37,9 @@
     <% if policy(@draw).intent_report? %>
       <p><%= link_to 'View intent report', intent_report_draw_path(@draw) %></p>
     <% end %>
+    <% if policy(@draw).bulk_on_campus? %>
+      <p><%= link_to 'Make all students on-campus', bulk_on_campus_draw_path(@draw), method: :patch, class: 'button' %></p>
+    <% end %>
```

Comment:
also, should we add a descriptive [tooltip](http://foundation.zurb.com/sites/docs/v/5.5.3/components/tooltips.html)?

I think this one should say "Sets all students who haven't declared their housing intent to on campus"